scripts/clean-db-dump.sql

```diff
@@ -112,6 +112,38 @@ WHERE declaration IS NOT NULL
   AND declaration::varchar != 'null'
   AND declaration::varchar != '{}';
 
+-- Create some fake draft and closed brief responses on closed briefs using eligible suppliers
+-- Ensuring essential requirements evidence length matches that of the brief essential requirements
+INSERT INTO brief_responses (git push 0-family()
+      data,
+      brief_id,
+      supplier_id,
+      created_at,
+      submitted_at
+)
+SELECT
+    ('{"essentialRequirements": [' || rtrim(repeat('{"evidence": "Some super good evidence."}, ', essential_requirements_len), ', ') || '], "niceToHaveRequirements": [{"yesNo": false}, {"yesNo": true, "evidence": "No jump is too high."}, {"yesNo": true, "evidence": "NEIGH"}], "availability": "09/09/17", "essentialRequirementsMet": true, "respondToEmailAddress": "example-email@gov.uk"}')::json,
```

Comment:
Nice on the essential requirements length. Can we also do it for the nice to have requirements as they can be off various length?

Reply:
Also is there anyway to split this line over multiple lines to improve the readability whilst still writing valid SQL?

Reply:
Done 👍